zilencer/migrations/0032_remotepushdevicetoken_backfill_ios_app_id.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """
+    Previous versions of zilencer dropped the ios_app_id on the floor
+    when registering a push token, and then effectively assumed
+    the value was "org.zulip.Zulip".
+
+    We're going to start actually using that parameter.  To preserve
+    the existing behavior for existing tokens, backfill the value.
+    """
+
+    dependencies = [
+        ("zilencer", "0031_alter_remoteinstallationcount_remote_id_and_more"),
+    ]
+
+    FORMERLY_IMPLICIT_IOS_APP_ID = "org.zulip.Zulip"
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                (
+                    """
+            UPDATE zilencer_remotepushdevicetoken
+            SET ios_app_id = %s
+            WHERE kind = 1 AND ios_app_id IS NULL
+            """,
+                    [FORMERLY_IMPLICIT_IOS_APP_ID],
```

Comment:
I don't have a good handle on how expensive this migration query might be. It's pretty simple, and purely SQL; but in the production bouncer it's a fair number of rows. This version does it the direct way as a single query, but I'd appreciate thoughts on whether that's too much and it needs batching.

It'd also be possible to skip the migration, and have the code that consumes this just interpret NULL/None as the legacy value. But that seems less clean as an end state.

Reply:
I'd prefer to do the migration. @alexmv for thoughts on performance but fundamentally, we're the only once who will ever need to run the migration.

Reply:
I think this will be fine.  We could batch it, but I think it's probably small enough to just go for it.